main/models.py

```diff
@@ -1,3 +1,61 @@
 from django.db import models
+from django.contrib.auth.models import User
 
-# Create your models here.
+class Landlord(models.Model):
+    first_name = models.CharField(max_length=50)
+    last_name = models.CharField(max_length=50)
+    properties = models.ManyToManyField('Property')
+
+class Complex(models.Model):
+    name = models.CharField(max_length=150)
+    landlord = models.ForeignKey(Landlord)
+
+class Property(models.Model):
+    address = models.CharField(max_length=120)
+    city = models.CharField(max_length=100)
+    state = models.CharField(max_length=2)
+    zip_code = models.CharField(max_length=5)
+    latitude = models.DecimalField(max_digits=9, decimal_places=6)
+    longitude = models.DecimalField(max_digits=9, decimal_places=6)
+    furnished = models.BooleanField()
+    rooms = models.PositiveSmallIntegerField()
+    complx = models.ForeignKey(Complex)
```

Comment:
This did not, I have pushed a new commit that allows a property to not have an associated complex.